README.md

```diff
@@ -31,8 +31,8 @@ yarn add react-native-keyboard-aware-scroll-view
 ```
 
 ## Usage
-You can use the `KeyboardAwareScrollView`, the `KeyboardAwareListView` or the `KeyboardAwareFlatList`
-components. They accept `ScrollView`, `ListView` and `FlatList` default props respectively and
+You can use the `KeyboardAwareScrollView`, the `KeyboardAwareListView`, `KeyboardAwareSectionView` or the `KeyboardAwareFlatList`
+components. They accept `ScrollView`, `ListView`, `SectionView` and `FlatList` default props respectively and
```

Comment:
The component is `SectionList` and `KeyboardAwareSectionList` not `SectionView`.